README.md

```diff
@@ -4,35 +4,47 @@
 
 ## Building the CLI
 
-If you simply want to build a binary locally, run
-```sh
-$ make build/turnkey
-```
-
-We use [GoReleaser](https://goreleaser.com/) to build and release our binaries.
-
-To build a release locally:
+If you simply want to build binaries locally, run
 ```
-$ goreleaser release --snapshot --rm-dist
+$ make
 ```
 
 To release:
 ```
-# Generate a Github token with "write:packages"
-# ==> https://github.com/settings/tokens
-$ export GITHUB_TOKEN=<your token>
-$ git tag -s -a vx.y.z -m "New release: x.y.z" # create a signed tag
-$ git tag -v vx.y.z # verify the tag
-$ git push origin vx.y.z
-$ goreleaser release --rm-dist
+make dist
 ```
 
 ## Installing the CLI
 
-```
-brew tap tkhq/tap
-brew install turnkey
-```
+1. Clone repo
+    ```
+    git clone https://github.com/tkhq/tkcli
+    cd tkcli
+    ```
+
+2. Review source
+    * Ideal: Review of the entire supply chain is recommended for high risk uses
+    * Minimal: review the "attest" "sign" and "verify" targets in the Makefile
+
+3. Review binaries
+    * Ideal: ```make attest sign```
+        * Reproduce and sign binaries yourself
+	* We welcome PRs with external verification signatures
+    * Recommended: ```make attest```
+        * Prove published source code matches pubished binaries
+    * Minimal: ```make verify```
+        * Prove multiple people signed binaries
+        * Ensure signatures are by people whose reputations you trust
+
+4. Install binary
+
+    Replace "linux" and "amd64" with your preferred architecture:
+
+    ```
+    mkdir -p ~/.local/bin
+    cp tkcli/dist/turnkey.linux.amd64 ~/.local/bin/turnkey
+    chmod +x ~/.local/bin/turnkey
+    ```
```

Comment:
We should be more explicit for macOS users and have a set of commands to copy/paste, ready to go. Bonus points for a cross-platform make target abstracting this (`make install`)